pydbzengine/__init__.py

```diff
@@ -18,10 +18,12 @@
 from jnius import autoclass
 from jnius import PythonJavaClass, java_method, JavaMethod
 
-################# CLASSES #################
+################# JAVA REFLECTION CLASSES #################
 Properties = autoclass('java.util.Properties')
 DebeziumEngine = autoclass('io.debezium.engine.DebeziumEngine')
+# following class is used by DebeziumEngine(above class)
 DebeziumEngineBuilder = autoclass('io.debezium.engine.DebeziumEngine$Builder')
+# NOTE: Override the notifying method somehow jnius was calling wrong java method. (first one)
 DebeziumEngineBuilder.notifying = JavaMethod(
     '(Lio/debezium/engine/DebeziumEngine$ChangeConsumer;)Lio/debezium/engine/DebeziumEngine$Builder;')
 StopEngineException = autoclass('io.debezium.engine.StopEngineException')
```

Comment:
reflection classes, and fix overloaded method. so when we pass Pyhon implementation of a java interface right method is called.